dropwizard-logging/src/main/java/io/dropwizard/logging/FileAppenderFactory.java

```diff
@@ -260,21 +260,20 @@ public boolean isValidFileConfiguration() {
 
                 return appender;
             } else {
-                final TimeBasedFileNamingAndTriggeringPolicy<E> triggeringPolicy;
+                final TimeBasedFileNamingAndTriggeringPolicy<E> triggeringPolicy = new DefaultTimeBasedFileNamingAndTriggeringPolicy<>();
+                final TimeBasedRollingPolicy<E> rollingPolicy;
                 if (maxFileSize == null) {
-                    triggeringPolicy = new DefaultTimeBasedFileNamingAndTriggeringPolicy<>();
+                    rollingPolicy = new TimeBasedRollingPolicy<>();
                 } else {
-                    final SizeAndTimeBasedFNATP<E> maxFileSizeTriggeringPolicy = new SizeAndTimeBasedFNATP<>();
-                    maxFileSizeTriggeringPolicy.setMaxFileSize(new FileSize(maxFileSize.toBytes()));
-                    triggeringPolicy = maxFileSizeTriggeringPolicy;
```

Comment:
Please declare the variable as `final` for consistency with rest of the codebase.